lib/packwerk/cli.rb

```diff
@@ -67,7 +71,9 @@ def execute_command(args)
           Subcommands:
             init - set up packwerk
             check - run all checks
-            update-deprecations - update deprecated references
+            update-todo - update package_todo.yml files
+            update - Shorthand for update-todo
+            update-deprecations - update package_todo.yml files (deprecated, use todo instead)
```

Comment:
I'm not totally sure what those are used but it might be worth auditing at some point 🤔 I think the former is used to basically find old violations that should be removed. Note sure about `generate_configs` without digging in further.